rpm/buteo-syncfw-qt5.spec

```diff
@@ -112,6 +112,7 @@ Summary: QML plugin for %{name}
 
 %prep
 %setup -q
+mkdir -p doc/html
```

Comment:
Ideally the build system should take of details needed for build, so something in the doc.pro/.pri direction would be more appropriate.

Reply:
It can't be in .pro or .pri, because it has to exist before qmake is run.

Reply:
The reason being that when qmake reads https://github.com/sailfishos/buteo-syncfw/blob/c772f0b5b6d507a2a6c915ab02ccf86261b1a423/doc/doc.pri#L12 it erroneously assumes that `html`is a file. Which is really stupid because it has '/' in the end...

Reply:
Installing files or from directories that didn't exist when the build started is nothing too special and any build system should really cope with that. Supposedly done million times already with qmake too, it's just how to use it.

On quick test a one character fix could be changing html/ to html/*. Debatable whether those should go to html subdir also in the target, but now the main doc/ is anyway empty so the html could be considered sort of redundant.

Reply:
> On quick test a one character fix could be changing html/ to html/*

I did actually consider it, but didn't try because.. well, I guess because it felt too obvious or something. Like that it can't work or it would have been used already.

> Debatable whether those should go to html subdir also in the target, but now the main doc/ is anyway empty so the html could be considered sort of redundant.

Yup. We can move it later if some content for the main doc dir appears from somewhere.